### PR TITLE
 API-based webapp

### PR DESCRIPTION
**Pull request by danutibe07**
An API web app that displays data about different series.
- [ ] Send and receive data from an API.
- [ ] Used webpack to build and start the app.
- [ ] Made use of callbacks and promises.
- [ ] Wrote unit tests to cover every aspect of the app.

### CODE REVIEW

File: test/api.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    expect(data.title).toEqual('delectus aut autem');
+  });
+
+  test('Posting data to the API', async () => {
+    const post = await postMovieMock(1, 'Hello World', false);
+    expect(post.title).toEqual('Hello World');
+    expect(post.userId).toEqual(1);
+    expect(post.completed).toBeFalsy();
+  });
+});
```
Comment (VICTOR-CHUKWU) [inline test/api.test.js:15]: - [ ] some of your tests are failing kindly fix this
image

File: src/module/displaySeries.js [hunk 2]
```diff
@@ -8,13 +8,19 @@ const displaySeries = (data) => {
...
-export default displaySeries;
\ No newline at end of file
+export default displaySeries;
+
+// const commentBtnfnx = () =>{
+//   const commentbtn = document.querySelectorAll("#comment")
+//   commentbtn.forEach((e) => {
+//       console.log("comment");
+//   });
+// }
```
Comment (VICTOR-CHUKWU) [inline src/module/displaySeries.js:26]: - [ ] kindly remove comments from your code base

File: src/index.html [hunk 3]
```diff
@@ -4,11 +4,37 @@
...
+      <div id="logo">
+        <a href="#"><img id="imageDiv" alt="SciFi logo" /></a>
+      </div>
+      <nav class="navbar">
+        <ul class="navlist">
+          <li class="list">Home</li>
+          <li class="list">About Us</li>
+          <li class="list">Contact</li>
+        </ul>
+      </nav>
```
Comment (VICTOR-CHUKWU) [inline src/index.html:34]: - [ ] there  should be counter present in the home page that show the number of items

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @danutibe07  and @Kaghenimbale  and @Rana-Mudassir , 
## STATUS: CHANGES REQUIRED :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/module/CommentTest/getComment.test.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+/**
```
Comment (huseyinbiyik) [inline src/module/CommentTest/getComment.test.js:1]: [OPTIONAL] Unfortunately, your tests were not built properly. You should write tests for the counter functions. But you are creating some functions and testing them which is not useful for testing the functionality of the app. 
Screenshot 2023-03-25 at 2 56 22 AM

**Review by huseyinbiyik — APPROVED** [review]
Hi team

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Git flow followed 👌
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
- Your files are not organized, tests and modules are located in different folders. Kindly organize your folder system to have a professional repository 👍 
-  Please make sure your commit messages have imperative tense. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**